client/spring-zeebe/src/main/java/io/camunda/zeebe/spring/client/bean/value/ZeebeDeploymentValue.java

```diff
@@ -8,11 +8,14 @@
 
   private List<String> classPathResources;
 
+  private List<String> bpmnLocations;
+
   private ClassInfo beanInfo;
 
-  private ZeebeDeploymentValue(List<String> classPathResources, ClassInfo beanInfo) {
+  private ZeebeDeploymentValue(List<String> classPathResources, ClassInfo beanInfo, List<String> bpmnLocations) {
```

Comment:
I think it should be either/or - not both attributes. And "classPathResources" could be translated into "resources" as I tried to mention in the overall comment